common/convert/convert.go

```diff
@@ -222,6 +222,15 @@ func (f *StringToFloat64) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// MarshalJSON implements the json.Marshaler interface.
+func (f StringToFloat64) MarshalJSON() ([]byte, error) {
+	if f == 0 {
+		return []byte(jsonStringIdent + jsonStringIdent), nil
+	}
```

Comment:
Hope no one ever wants to send "0" (but they have other options anyway)

Reply:
Panic() is an option.